extend/component/implementation/python.md

```diff
@@ -45,40 +45,86 @@ it is immediately written to the output file. This approach keeps only a single
+[Keboola Common Interface](https://developers.keboola.com/extend/common-interface/). It simplifies all tasks related
+ to the communication of the [Component](https://developers.keboola.com/extend/component/) with 
+ the Keboola Connection that is defined by the Common Interface. Such tasks are config manipulation, validation, 
+ component state, I/O handling, I/O metadata and manifest files, logging, etc.
+ 
+ **NOTE:** That this package is a replacement for the previous legacy [Python docker application](https://github.com/keboola/python-docker-application)
+
+The `CommonInterface` class provides following methods:
+
+- read and parse the configuration file and parameters: `configuration` object and `configuration.parameters` properties.
+- list input files and tables represented by Python objects for easier manipulation: 
+- work with [manifests](/extend/common-interface/manifest-files/) containing table and file metadata: `get_table_manifest()`, `get_file_manifest()`, `write_table_manifest()`, `write_file_manifest()` methods.
+- list expected outputs: `configuration.files_input_mapping`, `configuration.tables_input_mapping` properties.
 
-- read and parse the configuration file and parameters: `config_data` property and `get_parameters()` method.
-- list input files and tables: `get_input_files()`, `get_input_tables()` methods.
-- work with manifests containing table and file metadata: `get_table_manifest()`, `get_file_manifest()`, `write_table_manifest()`, `write_file_manifest()` methods.
-- list expected outputs: `get_expected_output_files()` and `get_expected_output_tables()` methods.
-
-Additionally, it also defines KBC's [CSV dialect](https://docs.python.org/3/library/csv.html#csv-fmt-params)
-to shorten up the CSV manipulation code.
 The library is a standard Python package that is available by default in the production environment.
-It is [ready for use on GitHub](https://github.com/keboola/python-docker-application), so it can be installed
-locally with `pip3 install https://github.com/keboola/python-docker-application/zipball/master`.
-A generated [documentation](https://github.com/keboola/python-docker-application/blob/master/doc/keboola.docker.html)
+It is a public PYPI project [keboola.component](https://pypi.org/project/keboola.component/), so it can be installed
+locally with `pip3 install keboola.component`.
+
+A generated [API documentation](https://htmlpreview.github.io/?https://raw.githubusercontent.com/keboola/python-component/main/docs/api-html/component/interface.html)
 is available for the package, and an actual working example can be found in our
-[sample component](https://github.com/keboola/python-custom-application-text-splitter/blob/master/main.py).
-Also note that the library does no special magic, it is just a mean to simplify things a bit for you.
+[Python template](https://bitbucket.org/kds_consulting_team/kbc-python-template/src/master/src/component.py).
```

Comment:
would be cool to merge this with https://github.com/keboola/component-generator/tree/master/templates